DatabaseWorkshop.md

```diff
@@ -0,0 +1,185 @@
+# Adding persistance to ToDoBackend with Swift-Kuery-ORM
+
+<p align="center">
+<img src="https://www.ibm.com/cloud-computing/bluemix/sites/default/files/assets/page/catalog-swift.svg" width="120" alt="Kitura Bird">
+</p>
+
+<p align="center">
+<a href= "http://swift-at-ibm-slack.mybluemix.net/"> 
+    <img src="http://swift-at-ibm-slack.mybluemix.net/badge.svg"  alt="Slack"> 
+</a>
+</p>
+
+## Prerequisite
```

Comment:
Given that the FoodTracker tutorial uses the format Pre-Requisites: it would be nice to keep the headings the same here.

Reply:
✅